### PR TITLE
support running without client certificate

### PR DESCRIPTION
**Pull request by ShakoHo**
@MikeLien @askeing @ypwalter r?
- add enable the option for firefox/chrome profile creation
- add checking with client certificate within PyDriveUtil.py
- move test_target to create test target function

### CODE REVIEW

File: configs/chrome/default.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "enable_create_new_profile": true,
```
Comment (ypwalter) [inline configs/chrome/default.json]: I think this should be applied to all config json files under `configs/chrome/` and `configs/firefox/`?

File: lib/common/pyDriveUtil.py [hunk 2]
```diff
@@ -12,8 +13,12 @@ class PyDriveUtil(object):
...
         if settings is None:
             settings = {"settings_file": None, "local_cred_file": "mycreds.txt"}
-        self.gauth = self.get_gauth(settings)
-        self.drive = GoogleDrive(self.gauth)
+        if os.path.exists(settings['local_cred_file']):
+            self.gauth = self.get_gauth(settings)
+            self.drive = GoogleDrive(self.gauth)
+        else:
+            raise Exception(
+                "Your current working dir didn't inculde the client certificate file [%s], please make sure the firefox/chrome profile creation is turned off, online mode is disabled and not running the test cases of GSuite and FB!")
```
Comment (ypwalter) [inline lib/common/pyDriveUtil.py]: Your current working dir `doesn't` `include` the client certificate file [%s]`.` Please make sure firefox/chrome profile creation is turned off, online mode is disabled`,` and `GSuite and FB cases are not test targets!`

File: lib/common/pyDriveUtil.py [hunk 3]
```diff
@@ -12,8 +13,12 @@ class PyDriveUtil(object):
     def __init__(self, settings=None):
         if settings is None:
             settings = {"settings_file": None, "local_cred_file": "mycreds.txt"}
-        self.gauth = self.get_gauth(settings)
-        self.drive = GoogleDrive(self.gauth)
+        if os.path.exists(settings['local_cred_file']):
+            self.gauth = self.get_gauth(settings)
```
Comment (ypwalter) [inline lib/common/pyDriveUtil.py]: `self.gauth` is not used elsewhere, I think you can use gauth instead like the refactor you made in:
https://github.com/Mozilla-TWQA/Hasal/pull/654/files#diff-9a13e7681f813c5a68e0ed49a6c4935fR89

**Comment by ShakoHo** [comment]
fix #654

**Pull request merged**